main.tf

```diff
@@ -0,0 +1,69 @@
+data "aws_caller_identity" "current" {
+}
```

Comment:
This dangling bracket will never fail to irritate me

Reply:
submit a pr for tf fmt. I'm sure they'll accept it!